src/server/dragonfly_test.cc

```diff
@@ -428,20 +428,25 @@ TEST_F(DflyEngineTest, StickyEviction) {
     ASSERT_EQ("OK", Run({"set", key, tmp_val}));
   }
 
-  for (ssize_t i = 0; i < 5000; ++i) {
+  bool done = false;
+  for (ssize_t i = 0; !done && i < 5000; ++i) {
     string key = StrCat("key", i);
-    auto set_resp = Run({"set", key, tmp_val});
-    auto stick_resp = Run({"stick", key});
+    while (true) {
```

Comment:
instead you can run in one command "set key val stick"

Reply:
i didn't know we could do this! this is the best solution!

Reply:
Me neither, and best indeed! :)